pages/unlock-publishing.tsx

```diff
@@ -0,0 +1,254 @@
+                  </Label>
+                  <TextInput
+                    type={"text"}
+                    id={"department"}
+                    name={"department"}
+                    className="required w-[34rem]"
+                    required
+                  />
+                </div>
+
+                <div className="focus-group">
+                  <Label id={"label-goals"} htmlFor={"goals"} className="required" required>
+                    {t("unlockPublishing.form.field3.title")}
+                  </Label>
+                  <TextArea
+                    id={"goals"}
+                    name={"goals"}
+                    className="required w-[34rem] mt-4"
+                    required
+                    characterCountMessages={{
+                      part1: t("formElements.characterCount.part1"),
+                      part2: t("formElements.characterCount.part2"),
+                      part1Error: t("formElements.characterCount.part1-error"),
+                      part2Error: t("formElements.characterCount.part2-error"),
+                    }}
+                  />
+                </div>
+
+                <div className="flex mt-14">
+                  <Button
+                    type="submit"
+                    className={` 
+                        mr-8
+                        bg-blue-dark text-white-default border-black-default py-4 px-8 rounded-lg border-2 border-solid
+                        hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                        focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+                      `}
+                    disabled={submitting}
+                  >
+                    {t("submitButton", { ns: "common" })}
+                  </Button>
+                  <StyledLink
+                    href={`/${i18n.language}/myforms/`}
+                    className={` 
+                          no-underline visited:text-black-default 
+                          bg-white-default text-black-default border-black-default py-4 px-8 rounded-lg border-2 border-solid
+                          hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                          focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+                        `}
+                  >
+                    {t("unlockPublishing.skipStepButton")}
+                  </StyledLink>
+                </div>
+              </form>
+            </>
+          )}
+        </Formik>
+      )}
+      {hasRequestedPublishing && (
+        <>
+          <h1>{t("unlockPublishingSubmitted.title")}</h1>
```

Comment:
Future Todo: Currently the page updates and is announced which works.  But adding a tabindex=-1 and focussing the H1 on load would make it really clear the page was updated. Though this may take some work and also other places in the app may want to follow this. Probably best to do when refactoring.